cmake/Macros.cmake

```diff
@@ -383,7 +383,7 @@ function(sfml_add_test target SOURCES DEPENDS)
     source_group("" FILES ${SOURCES})
 
     # create the target
-    add_executable(${target} ${SOURCES})
+    add_executable(${target} ${SOURCES} ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/../test/main.cpp)
```

Comment:
```suggestion
    add_executable(${target} ${SOURCES} ${PROJECT_SOURCE_DIR}/test/main.cpp)
```

Reply:
main.cpp? Looks to me like it's added correctly, and the tests would fail without it